easybuild/easyconfigs/h/HDF5/HDF5-1.10.2-PGI-18.4-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,26 @@
+name = 'HDF5'
+version = '1.10.2'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'PGI', 'version': '18.4-GCC-6.4.0-2.28'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = [
+    'bfec1be8c366965a99812cf02ddc97e4b708c1754fccba5414d4adccdc073866',  # hdf5-1.10.2.tar.gz
+]
+
+buildopts = 'CXXFLAGS="-noswitcherror"'
```

Comment:
@SimonPinches As mentioned in the other (now hidden discussion): this needs to be changed to avoid overwriting the `$CXXFLAGS` set by EasyBuild.
Also included a comment to clarify why `-noswitcherror` is needed is probably useful:

```python
# add -noswitcherror to avoid that libtool adds -pthread (which is not known by PGI compiler)
buildopts = 'CXXFLAGS="$CXXFLAGS -noswitcherror"'
```

Reply:
This is now done in a new commit.

Reply:
Hmm, seems like this isn't working as expected, I'm getting the exact problem you are trying to dance around (see incoming test report).

Reply:
Weird!